clear/main.py

```diff
@@ -550,6 +551,25 @@ def route(prefix, vrf, namespace):
 helper = util_base.UtilHelper()
 helper.load_and_register_plugins(plugins, cli)
 
+@click.option('--all', '-a', is_flag=True, help='Delete also compressed logs')
+@cli.command()
+def logging(all):
+    """Clear logging files"""
+    if os.path.exists("/var/log.tmpfs"):
+        log_path = "/var/log.tmpfs"
+    else:
+        log_path = "/var/log"
+    
+    if all:
+        files_to_delete = glob.glob("{}/syslog*".format(log_path))
```

Comment:
Best practice is to use the updated (and cleaner) python format string:
`files_to_delete = glob.glob(f"{log_path}/syslog*")`